ui/app.js

```diff
@@ -4693,51 +4685,36 @@ $(document).ready(() => {
-          );
-        }
+        // If the associated field is not checked, then populate the recommended fine and sentence fields
+        const associatedIncidentsContainer = (value.associated != 1) && `
+          <div class="associated-incidents-user-holder" data-name="${cid}" ></div>
+          <div class="manage-incidents-title-tag" data-id="${cid}">Recommended Fine</div>
+          <div class="associated-incidents-fine-input" data-id="${cid}"><img src="img/h7S5f9J.webp"> <input placeholder="0" disabled class="fine-recommended-amount" id="fine-recommended-amount" data-id="${cid}" type="number"></div>
+          <div class="manage-incidents-title-tag" data-id="${cid}">Recommended Sentence</div>
+          <div class="associated-incidents-sentence-input" data-id="${cid}"><img src="img/9Xn6xXK.webp"> <input placeholder="0" disabled class="sentence-recommended-amount" id="sentence-recommended-amount" data-id="${cid}" type="number"></div>
+          <div class="manage-incidents-title-tag" data-id="${cid}">Fine</div>
+          <div class="associated-incidents-fine-input" data-id="${cid}"><img src="img/h7S5f9J.webp"> <input placeholder="Enter fine here..." value="0" class="fine-amount" data-id="${cid}" type="number"></div>
+          <div class="manage-incidents-title-tag" data-id="${cid}">Sentence</div>
+          <div class="associated-incidents-sentence-input" data-id="${cid}"><img src="img/9Xn6xXK.webp"> <input placeholder="Enter months here..." value="0" class="sentence-amount" data-id="${cid}" type="number"></div>
+          <div class="associated-incidents-controls" data-id="${cid}">
+            <div id="jail-button" class="control-button" data-id="${cid}"><span class="fa-solid fa-building-columns" style="margin-top: 3.5px;"></span> Jail</div>
+            <div id="fine-button" class="control-button" data-id="${cid}"><span class="fa-solid fa-file-invoice-dollar" style="margin-top: 3.5px;"></span> Fine</div>
+            ${canSendToCommunityService ? `<div id="community-service-button" class="control-button" data-id="${cid}"> <span class="fa-solid fa-person-digging" style="margin-top: 3.5px;"></span>Community Service</div>` : ''}
+          </div>
+        `;
+
+        $(".incidents-ghost-holder").prepend(
+          `<div class="associated-incidents-user-container" data-id="${cid}">
+              <div class="associated-incidents-user-title">${value.name} (#${cid})</div>
+              <div class="associated-incidents-user-tags-holder">
+                  <div class="associated-incidents-user-tag ${warrantTag}" data-id="${cid}">Warrant</div>
+                  <div class="associated-incidents-user-tag ${guiltyTag}" data-id="${cid}">Guilty</div>
+                  <div class="associated-incidents-user-tag ${processedTag}" data-id="${cid}">Processed</div>
+                  <div class="associated-incidents-user-tag ${associatedTag}" data-id="${cid}">Associated</div>
+              </div>
+              ${associatedIncidentsContainer}

```

Comment:
The code here was refactored to reduce duplicated code. The only difference was that when we didn't want to show the fines+ times section, we had used display: none.

Now we will just conditionally render that section and can remove the duplicated code